src/openloco/windows/promptbrowsewnd.cpp

```diff
@@ -733,21 +732,43 @@ namespace openloco::ui::prompt_browse
             });
     }
 
-    static int sub_446F1D()
+    // 0x00446F1D
+    static bool filenameContainsInvalidChars()
     {
-        return call(0x00446F1D) & X86_FLAG_CARRY;
+        uint8_t numNonSpacesProcessed = 0;
+        for (char* ptr = &*_text_input_buffer; *ptr != '\0'; ptr++)
```

Comment:
`const char *`

the &* looks horrible but I know its just cause of the loco_global.

Reply:
Yeah, we should probably just redefine the `&` operator for `loco_global` as well.